src/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { ITypographyProps } from './types';
+
+export const Typography: React.FC<ITypographyProps> = ({ size, label }) => {
```

Comment:
also pass to h2 the rest of the props from the default heading interface youʼre extending from

Reply:
https://github.com/igorizviekov/mt-booking_app/pull/20/commits/32039294d6534ccbdc05c306ad374bcc95efda94